src/lib/pythongen.ml

```diff
@@ -236,10 +236,29 @@ let rec typecheck : type a.a typ -> string -> t list = fun ty v ->
     [
       Line (sprintf "if not (isinstance(%s, tuple) and len(%s) == 2):" v v);
       Block [ raise_type_error ];
-      Line (sprintf "left, right = %s" v)
+      Line (sprintf "v1, v2 = %s" v)
```

Comment:
Python linters may complain that the names are too short here. If that is the case just silence the warning :P

Reply:
Apparently they did not complain in the unit test.

Reply:
Good :)